tools/releaseBuild/azureDevOps/templates/release-CreateGitHubDraft.yml

```
@@ -16,6 +16,17 @@ steps:
     git clone https://$(AzureDevOpsPat)@mscodehub.visualstudio.com/PowerShellCore/_git/Internal-PowerShellTeam-Tools '$(Pipeline.Workspace)/tools'
   displayName: Clone Internal-Tools repository
 
+- pwsh: |
+    $Path = "$(System.ArtifactsDirectory)"
+    $OutputPath = Join-Path $Path ‘hashes.sha256’
+    $null = New-Item $OutputPath -ItemType File -Force
+    foreach ($file in Get-ChildItem -Path $Path -File ){
+        Get-FileHash -Algorithm SHA256 -Path $file |
+        ForEach-Object { "$($_.Hash) $($file.Name)" } |
```

```suggestion
        ForEach-Object { "$($_.Hash) *$($file.Name)" } |
```
add the * to indicate this is binary